### ISSUE TITLE
Schema validator does not throw error when nested object is not defined in schema

### ISSUE THREAD

**Issue by YourTechBud** [spaceuptech/space-cloud#429]
For the schema given below:
```graphql
type posts {
  _id: ID! @id
  title: String
  post: PostObj
}
```

`space-cloud` does not throw an error even if PostObj is not defined in the schema.

**Comment by YourTechBud** [comment]
@sharadregoti You think you can get this fixed?

**Comment by sharadregoti** [comment]
yes

**Comment by YourTechBud** [comment]
resolved in `v0.14.0`
